generic_expression_patterns_modules/ranking.py

```diff
@@ -817,3 +862,148 @@ def process_and_rank_genes_pathways(
     )
 
     return template_stats, simulated_summary_stats
+
+
+def format_enrichment_output(
+    local_dir, project_id, enrichment_method, pathway_names, num_runs
+):
+    """
+    This function formats the output from the different enrichment methods
+    to rank and summarize pathway results
+
+    1. GSVA, which returns a matrix that is
```

Comment:
This is helpful! Good to have a "key" for what each of the enrichment methods returns, and what you need to change to make them all look (roughly) the same for your analyses.